### ISSUE TITLE
Help messed up for Sln commands

### ISSUE THREAD

**Issue by KathleenDollard** [dotnet/cli#9440]
The commands portion of this help is messed up

```bash
$ dotnet sln -h
Usage: dotnet sln [options] <SLN_FILE> [command]

Arguments:
  <SLN_FILE>   Solution file to operate on. If not specified, the command will search the current directory for one.

Options:
  -h, --help   Show help information.

Commands:
  add <args>      .NET Add project(s) to a solution file Command
  list            .NET List project(s) in a solution file Command
  remove <args>   .NET Remove project(s) from a solution file Command
```

**Comment by livarcocc** [comment]
@KathleenDollard can you give us details on what is messed up? Also, what version of the SDK is this at?

**Comment by livarcocc** [comment]
cc @peterhuene who did a bunch of improvements for help.

**Comment by peterhuene** [comment]
This has been fixed in 2.1.4xx:

```
Usage: dotnet sln [options] <SLN_FILE> [command]

Arguments:
  <SLN_FILE>   The solution file to operate on. If not specified, the command will search the current directory for one.

Options:
  -h, --help   Show command line help.

Commands:
  add <PROJECT_PATH>      Add one or more projects to a solution file.
  list                    List all projects in a solution file.
  remove <PROJECT_PATH>   Remove one or more projects from a solution file.
```

**Comment by peterhuene** [comment]
@KathleenDollard Please re-open if you disagree with any of the 2.1.4xx text.  Thanks!
